server/common.ts

```diff
@@ -329,14 +330,14 @@ export function authenticateWithReject(request: express.Request, response: expre
 // For directly user facing endpoints
 export function authenticateWithRedirect(request: express.Request, response: express.Response, next: express.NextFunction) {
 	if (!request.isAuthenticated()) {
+		trackEvent("visit", request.ip);
```

Comment:
This will only track unauthenticated users accessing endpoints that require authentication (and that redirect instead of rejecting with a 400 error, i.e. non-API endpoints). Is that your intention? If you're trying to track all visits, maybe set up a super simple middleware that you attach to the main app in `app.ts`?